test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -111,9 +110,6 @@ def _get_latest_package_version(docker_exec_cmd, package, num_tries=3):
 
 @pytest.mark.model("N/A")
 @pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
-@pytest.mark.skipif(not is_mainline_context(),
```

Comment:
will this be reverted when merged?

Reply:
Yes, this will be reverted once this PR performs the safety check and verifies that the image built from the dockerfile does not have any vulnerabilities.

I've only manually tested by installing the latest cryptography version on an already built image.